Eat_and_Greet/src/screens/SettingsScreen/index.tsx

```diff
@@ -1,20 +1,60 @@
-import { Text, View } from 'react-native';
-import { StatusBar } from 'expo-status-bar';
+import React from 'react';
+import { View } from 'react-native';
+import { List, Switch } from 'react-native-paper';
+import { useNavigation } from '@react-navigation/native';
 
-import { Toolbar } from '../../comps/Toolbar/toolbar';
+const SettingsScreen: React.FC = () => {
+  const [notifications, setNotifications] = React.useState<boolean>(false);
 
-import styles from '../../style';
+  const toggleNotifications = () => {
+    setNotifications(!notifications);
+  };
+
+  const navigation = useNavigation();
+
+  return (
+    <View style={{ flex: 1, padding: 16, backgroundColor: '#7286D3' }}>
+      <View style={{ flex: 1, padding: 16, backgroundColor: '#E5E0FF' , borderRadius: 10}}>
+        
+        <List.Section>
+          <List.Subheader>General Settings</List.Subheader>
```

Comment:
I'd maybe change the fontSize of the List.Subheaders to be a bit bigger, but that's a personal preference. Just feel they may be hard to see if we add a bunch of other settings items.